scripts/installers/install-dotnet8

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Script to install dotnet 8
+
+set -e
+case ${TARGETARCH} in
+  arm64)
+    ARCH=arm64
+    CHECKSUM=3296d2bc15cc433a0ca13c3da83b93a4e1ba00d4f9f626f5addc60e7e398a7acefa7d3df65273f3d0825df9786e029c89457aea1485507b98a4df2a1193cd765
```

Comment:
Could we extract the checksums and the versions so that those versions are stored in the dockerfile? Otherwise changing them in the scripts won't cause a dockerfile rebuild.

Reply:
Good idea. Done.